hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -449,6 +499,26 @@ private void applyOperation(long counter) throws Exception {
           }
       );
       break;
+    case EMPTY_RPC:
```

Comment:
Do we need those? They're already available in `OmRPCLoadGenerator`.

Reply:
Removed